linux/ca-certificates/debian/build.gradle

```diff
@@ -108,8 +119,15 @@ task checkCaCertificatesDebian(type: Test) {
 	testClassesDirs = sourceSets.packageTest.output.classesDirs
 	classpath = sourceSets.packageTest.runtimeClasspath
 
+	def containerRegistry = getContainerRegistry()
```

Comment:
recommend usesContainerRegistry

Reply:
But this function actually gets the container registry that we are going to use. The specified one or the default one.